README.md

```diff
@@ -2403,6 +2406,52 @@ and 'absent'.
 Name of the ospf router. Valid value is a string.
 
 --
+### Type: cisco_ospf_area
+
+Manages an area for an OSPF router.
+
+| Platform | OS Minimum Version | Module Minimum Version |
+|----------|:------------------:|:----------------------:|
+| N9k      | 7.0(3)I3(1)        | 1.4.0                  |
+| N3k      | 7.0(3)I3(1)        | 1.4.0                  |
+| N5k      | 7.3(0)N1(1)        | 1.4.0                  |
+| N6k      | 7.3(0)N1(1)        | 1.4.0                  |
+| N7k      | 7.3(0)D1(1)        | 1.4.0                  |
+| N8k      | 7.3(0)F1(1)        | 1.4.0                  |
+
+#### Parameters
+
+##### `ensure`
+Determines whether the config should be present or not on the device. Valid values are 'present' and 'absent'.
+
+##### `authentication`
+Enables authentication for the area. Valid values are 'clear_text', 'md5' or 'default'.
+
+##### `default_cost`
+Default_cost for default summary Link-State Advertisement (LSA). Valid values are integer or keyword 'default'.
+
+##### `filter_list_in`
+Filter networks sent to this area. Valid values are string or keyword 'default'.
+
+##### `filter_list_out`
+Filter networks sent from this area. Valid values are string or keyword 'default'.
+
+##### `range`
+Configures an address range for an area.  Valid values are an array of [ip, not_advertise, cost] pairs or keyword 'default'. `ip` is mandatory in the sub-arrays but `not_advertise` and `cost` are optional.
+
+Example: `range => [['10.3.0.0/16', 'not_advertise', '23'],
+                    ['10.3.0.0/32', 'not_advertise'],
+                    ['10.3.0.1/32'],
+                    ['10.3.3.0/24', '450']]`
+
+##### `stub`
+Configures the area as a stub. Valid values are true, false or keyword 'default'.
+
+
+##### `stub_no_summary`
+Prevents Area Border Router (ABR) from sending summary LSAs into stub area. Valid values are true, false or keyword 'default'.

```

Comment:
This one is a bit problematic. The way I implemented is that, stub property MUST be set to true for the stub_no_summary to take affect. So we can have stub as true and stub_no_summary as true or false. Also stub can be false but we cannot have stub as false and stub_no_summary is true. 
If we want to use stub_no_summary in place of stub like you suggested here, then the NU has to change as well because I made stub as mandatory pre-requisite for stub_no_summary. Can you think this over once again and let me know?